src/tools/haxelib/Main.hx

```diff
@@ -977,11 +981,13 @@ class Main {
 				} catch( e : String ) {
 					throw "Error accessing Haxelib repository: $e";
 				}
-				return Path.addTrailingSlash( rep );
+				return [Path.addTrailingSlash( rep )];
 			} else
 				throw "This is the first time you are runing haxelib. Please run `haxelib setup` first";
 		}
 		rep = rep.trim();
```

Comment:
Seems like this line is unnecessary as rep is just going to be overwritten by the next two lines.